### PR TITLE
chore(deps): update angular to v15

### PR DESCRIPTION
**Pull request by mhaertwig**

### CODE REVIEW

**Review by jeripeierSBB — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -69,17 +69,17 @@
   },
   "devDependencies": {
     "@actions/cache": "3.0.6",
-    "@angular-devkit/build-angular": "15.0.0-rc.0",
-    "@angular-devkit/core": "15.0.0-rc.0",
-    "@angular-devkit/schematics": "15.0.0-rc.0",
+    "@angular-devkit/build-angular": "15.0.0",
+    "@angular-devkit/core": "15.0.0",
+    "@angular-devkit/schematics": "15.0.0",
     "@angular-devkit/schematics-cli": "14.2.6",
```
Comment (kyubisation) [inline package.json:75]: This should also be available as v15

File: package.json [hunk 2]
```diff
@@ -50,16 +50,16 @@
   "homepage": "https://angular.app.sbb.ch",
   "schematics": "./tools/schematics/collection.json",
   "dependencies": {
-    "@angular/animations": "15.0.0-rc.0",
+    "@angular/animations": "15.0.0",
     "@angular/cdk": "15.0.0-next.5",
```
Comment (kyubisation) [inline package.json:54]: Should this also be updated?

**Review by kyubisation — APPROVED** [review]
LGTM 👍 
A question about the other angular dependencies

**Pull request merged**